pkg/client/unversioned/clientcmd/client_config.go

```diff
@@ -358,21 +358,21 @@ func (inClusterClientConfig) ClientConfig() (*restclient.Config, error) {
 	return restclient.InClusterConfig()
 }
 
-func (inClusterClientConfig) Namespace() (string, error) {
```

Comment:
If it's part of an interface, then change the interface, too-- (string, bool, error) is a confusing return set.